src/Sulu/Component/Persistence/EventSubscriber/ORM/UserBlameSubscriber.php

```diff
@@ -0,0 +1,203 @@
+<?php
+/*
+ * This file is part of the Sulu CMF.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Component\Persistence\EventSubscriber\ORM;
+
+use Doctrine\ORM\Events;
+use Doctrine\ORM\EntityManager;
+use Doctrine\Common\EventSubscriber;
+use Doctrine\Common\Persistence\Event\LifecycleEventArgs;
+use Doctrine\Common\Persistence\Event\LoadClassMetadataEventArgs;
+use Sulu\Component\Persistence\Model\UserBlameInterface;
+use Symfony\Component\Security\Core\SecurityContextInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Sulu\Component\Security\UserInterface;
+use Symfony\Component\Security\Core\Authentication\Token\TokenInterface;
+
+/**
+ * Ensure that blame can be assigned to users when they break things.
+ *
+ * Persists the user that created and the last user that changed ORM classes
+ * implementing UserBlameInterface.
+ */
+class UserBlameSubscriber implements EventSubscriber
+{
+    const CHANGER_FIELD = 'changer';
+    const CREATOR_FIELD = 'creator';
+
+    /**
+     * @var ContainerInterface
+     */
+    private $container;
+
+    /**
+     * @var boolean
+     */
+    private $assignBlame;
+
+    /**
+     * NOTE: We inject the Container here because it is the only way currently to
+     *       avoid a cirular reference. Starting from Symfony2.6 we can use the
+     *       security.token_storage instead of the context, this will avoid the issue.
+     *
+     * @param SecurityContextInterface $securityContext
+     */
+    public function __construct(ContainerInterface $container, $assignBlame = false)
```

Comment:
Disabled automatic blame assignation until we decide what to do - we could remove the `$user` from all API's (e.g. Contact, Media).. or not. 

/cc @danrot @chirimoya @wachterjohannes